lib/skin_cache_helper.rb

```diff
@@ -0,0 +1,26 @@
+# note, if you modify this file you have to restart the server or console
+module SkinCacheHelper
+  def skin_cache_html_key(skin)
+    'skins_style_tags/v1/' + \
+    AdminSetting.default_skin.id.to_s + '/' + \
+    Skin.default.id.to_s + '/' + \
+    skin_cache_value(skin)
+  end
+
+  def skin_cache_footer_key
+    'Skins_menu/v1/' + \
+    (Rails.cache.fetch('skins_generation/site_skin')|| 0).to_s
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Surrounding space missing for operator `||`.